docs/metrics.md

```diff
@@ -0,0 +1,14 @@
+# Build Controller Metrics
+
+The Build component exposes several metrics to help you monitor the health and behavior of your build resources.
+
+Following build metrics are exposed at service `build-operator-metrics` on port `8383`.
+
+| Name | Type | Description | Status |
+| ---- | ---- | ----------- | ------ |
+| `up_status` | Gauge | The up status of the build-controller. | experimental |
+| `build_count` | Gauge | Build creation count. | experimental |
+| `buildrun_count` | Gauge | BuildRun execution count. | experimental |
```

Comment:
ditto

Reply:
Same as above ^^

Reply:
Fixed all names as discussed, resolve the conversations. Thanks!